drivers/io/io_fip.c

```diff
@@ -341,7 +414,7 @@ int register_io_dev_fip(const io_dev_connector_t **dev_con)
 	int result;
 	assert(dev_con != NULL);
 
```

Comment:
Need a comment here: 
```
  /*
   * Since dev_info isn't really used in io_register_device, always
   * use the same device info at here instead.
   */
```

Reply:
ok